modules/readme/Makefile.docs.repo

```diff
@@ -0,0 +1,19 @@
+
+define repo-docs-provide
+--datasource repo=file:///$(TEMPLATES)/repo.yml
+endef
+
+define repo_data-docs-provide
+--datasource repo_data=file:///tmp/repo_data.yml
```

Comment:
I don't like the reliance on `/tmp` without a tmp directory. If we're usnig this in multiple projects, there will be collisions.